STYLE.md

```diff
@@ -99,6 +101,7 @@ following abbreviations we assume readers will already know.
 | preimage | pre-image | |
 | proof of work | proof-of-work | proof-of-work may be used as an adjective phrase (e.g. "Bitcoin's proof-of-work security is economic in nature"). | |
 | redeemScript | redeem script | |
+| scriptpath | script path or script-path | |
```

Comment:
I prefer "key path" and "script path" (and "fee rate").

Reply:
I think we're fairly committed on *feerate* (186 references verus 10 for *fee rate*).

I'll remove keypath and scriptpath from this PR so we can discuss them without time pressure.